### ISSUE TITLE
Support for dark mode

### ISSUE THREAD

**Issue by Bhupesh-V** [developersIndia/saadhan#30]
### Feature details?

A dark mode is a must mode!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by shreydd** [comment]
I'm guessing a re-work of the styling of the whole website is being re-made with bulma CSS in a different PR? I think it's best to club this PR with the other one as it will involve a major change in the whole codebase.

**Comment by siddh384** [comment]
hey, I am interested in working on this issue. Can you please assign it to me?
As far as I understand, I have to make the button/toggle switch that converts the page from light mode to dark mode, correct?
Does Bulma CSS need to be used for this issue or can I use Vanilla CSS?

**Comment by Bhupesh-V** [comment]
> hey, I am interested in working on this issue. Can you please assign it to me? As far as I understand, I have to make the button/toggle switch that converts the page from light mode to dark mode, correct? Does Bulma CSS need to be used for this issue or can I use Vanilla CSS?

Yes, correct, you can use vanilla CSS but make sure it plays nicely with bulma. Otherwise, you can have a look at ready-made solution like https://jenil.github.io/bulmaswatch/

**Comment by Dev-Arhaan** [comment]
Hi is this issue open to work on? I'd like to contribute

**Comment by Bhupesh-V** [comment]
> Hi is this issue open to work on? I'd like to contribute

You can take this up @Dev-Arhaan.

**Comment by Dev-Arhaan** [comment]
where is this site hosted? 

the main link redirects it to another site

**Comment by Bhupesh-V** [comment]
> where is this site hosted?
> 
> the main link redirects it to another site

Hey this should be fixed now, seems like the deployment was under auth.
